frontend/app/components/page-details.tsx

```diff
@@ -1,14 +1,11 @@
 import { useTranslation } from 'react-i18next';
 
-import { useBuildInfo, usePageIdentifier } from '~/utils/route-utils';
+import { getClientEnv } from '~/utils/env-utils';
+import { usePageIdentifier } from '~/utils/route-utils';
 
 export function PageDetails() {
   const { t } = useTranslation(['gcweb']);
-
-  const buildInfo = useBuildInfo() ?? {
-    buildDate: '2000-01-01T00:00:00Z',
-    buildVersion: '0.0.0',
-  };
+  const { BUILD_DATE: buildDate, BUILD_VERSION: buildVersion } = getClientEnv();
```

Comment:
Use `useClientEnv` hook.
```suggestion
  const { BUILD_DATE: buildDate, BUILD_VERSION: buildVersion } = useClientEnv();
```

Reply:
> think we should not extend server config with client config to have a good seperation of concerns.

I disagree with this.

Reply:
> Use useClientEnv hook.

Using a hook is unnecessary.

Reply:
Using that hook doens't required the config to be reparse again since they are provider by root loader. Use `useClientEnv` hook.